### PR TITLE
Add RobotFramework export option on the recorder

### PR DESCRIPTION
**Pull request by NeoMorfeo**
* Add the RobotFramework option to allow exporting the recorded session to a runable RobotFramework syntax, usign the variables and capabilities needed and obtained from caps

### CODE REVIEW

File: app/renderer/lib/client-frameworks/robot.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+*** Settings ***
+Library           AppiumLibrary
+
+*** Variables ***
+\$\{REMOTE_URL\}   ${this.serverUrl}
+${this.getCapsVariables}
+
+*** Test Cases ***
+Test case name
+${this.indent(this.getApplicationInitialization(),4)}
```
Comment (jlipps) [inline app/renderer/lib/client-frameworks/robot.js]: spacing here and next line (should be `, 4`)

File: app/renderer/lib/client-frameworks/robot.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  getPythonVal (jsonVal) {
+    if (typeof jsonVal === 'boolean') {
+      return jsonVal ? "True" : "False";
+    }
+    return jsonVal;
+  }
+
+  wrapWithBoilerplate (code) {
+    return `# This sample code uses the Appium robot client
+# pip install robotframework-appiumlibrary
```
Comment (jlipps) [inline app/renderer/lib/client-frameworks/robot.js]: is robot framework only in python? or should this be called "RobotFramework Python"?

File: app/renderer/lib/client-frameworks/robot.js (hunk 1 above)
Comment (NeoMorfeo) [reply app/renderer/lib/client-frameworks/robot.js]: Fixed on cefc256a875705a082bb2b1ac18451c5c3759103

**Comment by NeoMorfeo** [comment]
Yes, I made several tests and proofs using the recorder and use the exported code directly on robot.

Would you like I give some more info or test examples?

**Comment by dpgraham** [comment]
Looks good to me. I'm ready to merge if you are @jlipps

**Comment by dpgraham** [comment]
@NeoMorfeo we've added many new commands, if you'd like them added please update: https://github.com/appium/appium-desktop/tree/master/app/renderer/lib/client-frameworks

**Comment by NeoMorfeo** [comment]
Oh great! I will do for sure, let me read all the docs and implement it against robot!

**Pull request merged**